src/cpp-utils/crypto/cryptopp_byte.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#ifndef _CRYPTOPP_BYTE_H
```

Comment:
Like in the other header files you touched. Something like `MESSMER_CPPUTILS_CRYPTO_CRYPTOPPBYTE_H_` (although I should probably get rid of the `MESSMER` in all the include guards).